scripts/get-env-venue-owners.ts

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env node -r esm -r ts-node/register
+
+import fs from "fs";
+import { resolve } from "path";
+
+import admin from "firebase-admin";
+
+import {
+  initFirebaseAdminApp,
+  makeScriptUsage,
+  parseCredentialFile,
+} from "./lib/helpers";
+
+const usage = makeScriptUsage({
+  description: "Print venue owners' email addresses.",
+  usageParams: "[CREDENTIAL_PATH] filterArray",
+  exampleParams: "[theMatchingAccountServiceKey.json] gmail.com",
+});
+
+const [credentialPath, ...filterArray] = process.argv.slice(2);
```

Comment:
not `filterArray`, but `emailSubstrToIgnore` (or `substingsToExclude`, or similar)

because `filterArray` is not fully clear: include only those, which are in the list, or filter them out.

Reply:
fixed f0ac60e